ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -374,6 +374,29 @@ commonBuildDepends =
   , "hpc == 0.6.*"
   ]
 
+-- | This utility factored out to avoid repetion.
+libBinParserModules :: IO ([Cabal], [Cabal], [String])
+libBinParserModules = do
+    lib <- mapM readCabalFile cabalFileLibraries
+    bin <- (:[]) <$> readCabalFile cabalFileBinary
+    parserModules <- calcParserModules
+    return (lib, bin, parserModules)
+
+-- | Call this after cabal file generation. These files are generated
+-- from '.hsc' files in the source tree and we prefer to ship those in
+-- the sdists rather than these. If we don't remove them, some
+-- ambiguity results. Cabal and stack are OK with that but bazel gets
+-- messed up.
+removeGeneratedIntermediateFiles :: IO ()
+removeGeneratedIntermediateFiles = do
```

Comment:
How hard is it to automate this list? Can we just glob for `.hsc` files and remove all corresponding `.hs` files?

Reply:
Absolutely, I’m also fine with just ignoring this and not even raising an issue :slightly_smiling_face: